parser.js

```diff
@@ -766,13 +778,66 @@
         group.name = name;
         return group;
       }
+      else if (features.modifiersGroup && str.indexOf("(?") == pos && str[pos+2] != ":") {
```

Comment:
I prefer shorter option name, such as `features.modifiers`. But that's only my opinion.